libnmstate/nm/connection.py

```diff
@@ -145,6 +147,11 @@ def safe_activate_async(self):
 
         cancellable = self._mainloop.new_cancellable()
 
+        if self.nmdevice and self.nmdevice.get_nm_plugin_is_missing():
```

Comment:
What happens when the plugin is missing but the device does not exist? Does this case not happen?

Reply:
I think this can happen. I am trying to cover this in manual testing but if this happens I don't know how could we fix that case.